app/connectors/customJSConnector.js

```diff
@@ -0,0 +1,35 @@
+const requestSender = require('../helpers/requestSender'),
```

Comment:
most of this file is duplicated.

imo you should have two public function which just passes the url to new private function which executes the request with the headers and response logging

Reply:
done